nucleus/dataset_item.py

```diff
@@ -127,24 +119,19 @@ class DatasetItem:  # pylint: disable=R0902
     )
     metadata: Optional[dict] = None
     pointcloud_location: Optional[str] = None
-    upload_to_scale: Optional[bool] = True
```

Comment:
We sure that nobody is using this? Otherwise I'd keep this -> change it to None and add a warning if this is not None that you should create a privacy dataset.

Reply:
I checked all dataset items, no one uses this feature (it was also broken before too)